pyneuroml/plot/PlotMorphologyVispy.py

```diff
@@ -769,6 +832,9 @@ def plot_3D_cell_morphology(
     except Exception:
         axon_segs = []
 
+    cell = rotate_cell(cell, x_angle, y_angle, 0, 'yxz', False, False)
+    cell = translate_cell_to_coords(cell, False, [0,0,0])
```

Comment:
I assume the order of operations is important---would it be different if we:

- first translated the cell to origin
- did the pca
- rotated the cell about the soma (origin)?

Reply:
- first translated the cell to origin
- did the pca
These two steps can be done in any order ,I believe, without any change because the PCA first translates the points so that their center is at the origin and then applies other computations.

But after that, the best way is in fact to translate and then rotate, now that I think of it, because the rotations and the angles are calculated based on the PCA axis, which starts at the origin. So to apply them succesfully  to the cell, it should be close to the origin in order to have similar results.